recipe/meta.yaml

```diff
@@ -12,32 +12,34 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  skip: True  # [py<36]
```

Comment:
If this is a noarch python, `skip: True` shouldn't be applied/needed.

Reply:
I removed that line